statsmodels/stats/pscore_match.py

```diff
@@ -0,0 +1,242 @@
+'''Propensity Score Matchig
+
+
+Author: 
+License: BSD-3
+
+'''
+
+import statsmodels.api as sm
+import numpy as np
+from statsmodels.stats.weightstats import CompareMeans, DescrStatsW
+import pandas as pd
+from scipy import stats
+
+
+class PropensityScoreMatch(object):
+    def __init__(self, assigment_index, covariates, treatment_objective_variable,\
+                 algo = 'strata', radius = 0.01, regression = 'logit'):
+        self.assigment_index = assigment_index
+        self.covariates = covariates
+        self.treatment_objective_variable = treatment_objective_variable
+        if regression == 'logit':
```

Comment:
Better to use duck typing here?  We might have other binary regression models now or in the future.